Makefile

```diff
@@ -193,7 +193,7 @@ docclean:
 run-ci:
 	$(PYTHON) ./configure $(CONFIG_FLAGS)
 	$(MAKE)
-	$(MAKE) test-ci
+	$(MAKE) test-ci test-addons
```

Comment:
Sorry I copied that from my local changes which include https://github.com/nodejs/node/pull/2424 and add `internet`. I wanted to suggest adding build-addons to the rule prerequisites and addons to the test runner arguments. So you can try:

```
test-ci: build-addons
    $(PYTHON) tools/test.py -p tap --logfile test.tap --mode=release message parallel sequential addons
```